hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestReplicationSource.java

```diff
@@ -416,6 +416,17 @@ public synchronized UUID getPeerUUID() {
 
   }
 
+  public static class FaultyReplicationEndpoint extends DoNothingReplicationEndpoint {
```

Comment:
Bumping into this during rebase of HBASE-18070 branch. If count is not used, just remove it from this class?

Reply:
Yeah, I guess it's a left over from previous commits on this same PR.